src/test/java/com/gizasystems/automationexercise/tests/test/PlaceOrder_RegisterBeforeCheckout.java

```diff
@@ -0,0 +1,79 @@
+package com.gizasystems.automationexercise.tests.test;
+
+import com.gizasystems.automationexercise.pages.*;
+import com.shaft.driver.SHAFT;
+import io.qameta.allure.Epic;
+import io.qameta.allure.Feature;
+import io.qameta.allure.Story;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import org.testng.annotations.BeforeClass;
+
+@Epic("Automation Exercise")
+@Feature("Placing Order")
+@Story("Place Order : Register Before Checkout")
+
+public class PlaceOrder_RegisterBeforeCheckout {
+    private SHAFT.TestData.JSON testData;
+    private SHAFT.TestData.JSON productData;
+    private SHAFT.TestData.JSON CardData;
+    private SHAFT.GUI.WebDriver driver;
+    @BeforeClass
+    public void beforeClass() {
+        testData = new SHAFT.TestData.JSON("src/test/resources/testDataFiles/RegisterUser.json");
+        productData = new SHAFT.TestData.JSON("src/test/resources/testDataFiles/Product.json");
+        CardData = new SHAFT.TestData.JSON("src/test/resources/testDataFiles/CardData.json");
```

Comment:
We only need one test data json path as agreed, why do you have 3?

Reply:
To make all of them accessible to my TC as  I test a full Scenario

Reply:
We agreed to have 1 to 1 relationship between a test class and a test data json file to make the code and test classes more independent on each other's and enhance the readability and scalability. Please make sure to apply the conventions we agreed on.